pkg/proxy/iptables/proxier.go

```diff
@@ -606,7 +616,8 @@ func (proxier *Proxier) syncProxyRules() {
 			} else {
 				socket, err := openLocalPort(&lp)
 				if err != nil {
-					glog.Errorf("can't open %s, skipping this nodePort: %v", lp.String(), err)
+					proxier.recorder.Eventf(proxier.nodeRef, "FailedToOpenNodePort", "Can't open %s, skipping this nodePort", lp.String())
```

Comment:
This is the same - it's about the service.  well AND the node, but if I were looking to debug a service and this was happening I would want this on the service.